ERCS/erc-7579.md

```diff
@@ -218,9 +218,8 @@ Smart accounts MAY implement a fallback function that forwards the call to a fal
 
 If the smart account has a fallback handler installed, it:
 
-- MUST implement authorization control
```

Comment:
what about: `MUST implement authorization control and SHOULD use hooks to implement this`

Reply:
SHOULD or MAY?

instead of hooks we can also mention that 
> ensure authorization control is implemented in the fallback handler itself, based on the appended msg.sender

wdyt? @kopy-kat

Reply:
I think if hooks as to ban some selectors.

Reply:
but authorization control is implemented by fallback handlers itself right.

Reply:
yeah thats a good point